### PR TITLE
02k: Removed menu option Override non-electrified route line-voltage

### PR DESCRIPTION
**Pull request by cjakeman**
See https://blueprints.launchpad.net/or/+spec/update-menu-options
and http://www.elvastower.com/forums/index.php?/topic/35301-menu-options/

Can now run electric locos on non-electricified routes.
An information message is shown when the pantograph is raised.

### CODE REVIEW

File: Source/ORTS.Settings/UserSettings.cs [hunk 1]
```diff
@@ -247,8 +247,8 @@ public enum DirectXFeature
         public bool TunnelResistanceDependent { get; set; }
         [Default(false)]
         public bool WindResistanceDependent { get; set; }
-        [Default(false)]
-        public bool OverrideNonElectrifiedRoutes { get; set; }
+        //[Default(false)]
+        //public bool OverrideNonElectrifiedRoutes { get; set; }
```
Comment (Sharpe49) [inline Source/ORTS.Settings/UserSettings.cs:251]:
```suggestion
```

**Review by Sharpe49 — CHANGES_REQUESTED** [review]

File: Source/ORTS.Settings/UserSettings.cs (hunk 1 above)
Comment (cjakeman) [reply Source/ORTS.Settings/UserSettings.cs:251]: Thanks, Cedric. Missed that.

**Review by pzgulyas — DISMISSED** [review]
Seems good

**Review by Sharpe49 — APPROVED** [review]

**Review by Sharpe49 — CHANGES_REQUESTED** [review]
The merge was incomplete. You probably should rebase your branches instead of merging.

**Comment by cjakeman** [comment]
@Sharpe49 Hi Cedric,

Thanks for that.

I've overcome the conflicts, now, so it would be great if you could re-approve this PR.

Thanks,

Chris

**Review by Sharpe49 — APPROVED** [review]

**Pull request merged**